ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml

```diff
@@ -71,6 +71,17 @@ tests:
     version: "4.10"
   steps:
     workflow: openstack-k8s-operators-test-build-deploy
+- as: openstack-operator-build-deploy-kuttl
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.10"
+  optional: true
+  steps:
+    workflow: openstack-k8s-operators-test-build-deploy-kuttl
```

Comment:
in order to run kuttl workflow, you also need to add:
```
- build_args:
  - name: KUTTL_VERSION
    value: v0.15.0
  dockerfile_literal: |
    FROM openstack-operator-ci-build-deploy
    ARG KUTTL_VERSION=v0.15.0
    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
    RUN chmod +x ./kubectl-kuttl
    RUN mv ./kubectl-kuttl /usr/local/bin
    RUN yum install -y skopeo
  from: openstack-operator-ci-build-deploy
  to: openstack-operator-ci-build-deploy-kutt
```
to build the container image needed for kuttl run